agent/doc/swdesign/README.md

```diff
@@ -658,7 +659,7 @@ Status: approved
 
 When requested, the RuntimeFacade shall delete a workload by:
```

Comment:
The sentence sound incomplete to me. What about write it in the same way as the new requirement:

```suggestion
When the RuntimeFacade is requested to delete the workload, then the RuntimeFacade shall delete a workload by:
```